pkg/drivers/jetstream/jetstream.go

```diff
@@ -0,0 +1,992 @@
+package jetstream
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"regexp"
+	"sort"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/k3s-io/kine/pkg/drivers/jetstream/kv"
+	"github.com/k3s-io/kine/pkg/server"
+	"github.com/k3s-io/kine/pkg/tls"
+	"github.com/nats-io/jsm.go/natscontext"
+	"github.com/nats-io/nats.go"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	kineBucket             = "kine"
+	revHistory             = 12
```

Comment:
It would be nice if `bucketRevisionHistory` and `slowMethodMilliseconds` were added to the DSN so that users can configure these